clients/java/src/main/java/io/camunda/client/impl/CamundaClientImpl.java

```diff
@@ -796,14 +797,21 @@ public CreateDocumentCommandStep1 newCreateDocumentCommand() {
 
   @Override
   public DocumentContentGetRequest newDocumentContentGetRequest(final String documentId) {
-    return new DocumentContentGetRequestImpl(httpClient, documentId, null, config);
+    return new DocumentContentGetRequestImpl(httpClient, documentId, null, null, config);
```

Comment:
`DocumentContentGetRequest` should be extended to allow specifying the content hash. If the user creates the command using this method (only with document ID), it should be possible to specify content hash later in the builder.